app/src/components/JobCard.css

```diff
@@ -0,0 +1,5 @@
+.border-gradient {
+    border-image: linear-gradient(to right, #D799D5, #B5B9F1);
+    border-image-slice: 1;
+    border-radius: 0.75rem; 
+}
```

Comment:
I don't think you can use tailwind to apply a gradient to borders which is why I created this css file. However, I'm not sure where is the best place to put this file?